gcc/config/riscv/constraints.md

```diff
@@ -168,6 +171,23 @@
        (and (match_test "IN_RANGE (ival, 0, 1073741823)")
             (match_test "exact_log2 (ival + 1) != -1"))))
 
+(define_constraint "CVmp"
+  "An address for post-modify or reg+reg stores and loads"
+  (and (match_code "mem")
```

Comment:
Again, instead of `match_code "mem"`, reuse the default mem constraint `(match_operand 0 "TARGET_MEM_CONSTRAINT")`.

Reply:
reference to unknown predicate 'TARGET_MEM_CONSTRAINT'
I can't do this.

Reply:
I should have said `(match_operand 0 "memory_operand" "TARGET_MEM_CONSTRAINT")`, but then the predicate gets checked which is unnecessary and wasn't being done before. I'd just copy the conditions from the default in `common.md` then.